### ISSUE TITLE
[Bug Report][2.6.2] VNavigationDrawe ，attr expandOnHover always enable(true)

### ISSUE THREAD

**Issue by iam993** [vuetifyjs/vuetify#14555]
### Environment
**Vuetify Version:** 2.6.2
**Last working version:** 2.6.2
**Vue Version:** 2.6.9
**Browsers:** Chrome 89.0.4389.90
**OS:** Windows 10

### Steps to reproduce
/

### Expected Behavior
/

### Actual Behavior
/

### Reproduction Link
<a href="https://codepen.io/" target="_blank">https://codepen.io/</a>

### Other comments
/

**Comment by KareemDa** [comment]
@iam993 
What is the problem? any description? reproduction link?

**Comment by Ahriana** [comment]
im not sure if this the same issue but since the last release ive had the navigation draw push the content of the page perminantly likes its always expanded
![image](https://user-images.githubusercontent.com/28003629/147880668-f5b903a5-8303-458c-b307-1bac61a9444e.png)

i will look into seeing if i can repo it in code pen later

**Comment by iam993** [comment]
@KareemDa 
just like vuetify doc.
https://vuetifyjs.com/en/components/navigation-drawers/#mini-variant
when [:mini-variant.sync] is binded, expandOnHover is true

**Comment by iam993** [comment]
@Ahriana 
I also found the same problem, everything is normaly in ver 2.6.1

**Comment by Ahriana** [comment]
yeah ive just bypassed the padding at the moment until i have the time to look into it more `style="padding-left: 64px"`

**Comment by hadalin** [comment]
The problem could be missing [this](https://github.com/vuetifyjs/vuetify/commit/03e683f286e652b942b83a817cecc080d4c4a6eb#diff-98d92001e9cf0f207f45c674457cea37a833979a1b0f0b143cb3714454acdddeL369) `if (this.expandOnHover) {` [here](https://github.com/vuetifyjs/vuetify/commit/03e683f286e652b942b83a817cecc080d4c4a6eb#diff-98d92001e9cf0f207f45c674457cea37a833979a1b0f0b143cb3714454acdddeR354).

**Comment by Ahriana** [comment]
i dont think so as its applies earlier in the file instead, the behaviour is as if the side bar is always expanded, but the expand behaviour is always the same but the content is pushed over as if the minivariant is being ignored in style but not behaviour

**Comment by KaelWD** [comment]
Related: #13309

@Ahriana I don't see that, please open a new issue if you can reproduce it.

**Comment by Tofandel** [comment]
This is not fixed correctly, now if miniVariant is true and expandOnHover is false it will update miniVariant on hover anyways

**Comment by Tofandel** [comment]
Why was my comment hidden?
Anyways found out that you need to explicitely set `:permanent="true"` which is a redundant prop but anyways, this is what's needed now

```vue
    :expand-on-hover="expandOnHover"
    :permanent="!expandOnHover"
 ```

**Comment by L2Ernestoo** [comment]
> correctly, now if miniVariant
How did you make it work? I was trying with what you put but still the same.

![image](https://user-images.githubusercontent.com/42287457/151016473-a0651edd-2ce8-4a34-9056-0792d508eba5.png)

**Comment by theRenard** [comment]
I still have the issue, or maybe the doc needs an update because what was working in 2.5.10 is not working anymore with the same configuration

`    app
    expand-on-hover
    disable-resize-watcher
    hide-overlay
    permanent
    mini-variant-width="80px"`

**Comment by Tofandel** [comment]
What are you expecting vs what are you getting? I see no difference with the props you gave in 2.6.1 and 2.6.3, the bug in 2.6.2 seems to have been fixed in 2.6.3

**Comment by theRenard** [comment]
Expecting

<img width="421" alt="image" src="https://user-images.githubusercontent.com/61255863/151439068-5ecf7441-eb32-4159-a6b6-455c255dbc32.png">


Getting


![image (1)](https://user-images.githubusercontent.com/61255863/151439108-83b4b31f-bd81-41af-b23f-8a816e11cbbb.png)

We were experiencing this issue in production where CI wasn't honoring the yarn.lock file and the Vuetify version was the last available (2.6.3). We didn't have this issue on our local env where we had v.2.5.10 so we _fixed_ the issue bonding the package version to 2.5.10. We didn't had time to investigate more on this issue but if needed I can help to setup a codepen to reproduce it.

**Comment by KaelWD** [comment]
Please follow #14585
